topic_monitor/setup.py

```diff
@@ -1,17 +1,27 @@
-from ament_python.data_files import get_data_files
-from ament_python.script_dir import install_scripts_to_libexec
+import sys
+
 from setuptools import find_packages
 from setuptools import setup
 
 package_name = 'topic_monitor'
-data_files = get_data_files(package_name)
-install_scripts_to_libexec(package_name)
+
+# install scripts into libexec path
+if 'develop' in sys.argv:
+    sys.argv[sys.argv.index('develop') + 1:sys.argv.index('develop') + 1] = \
```

Comment:
> The Debian rules file will need to do the same as the build tool (pass a prefix but no script specific location) and the behavior should not change.

I'm not sure that this will work. Earlier iterations of the bloom changes passed only the prefix and that was not sufficient to put everything in the right place. I needed to pass both --install-lib and --install-scripts explicitly. Blooming this branch in a throwaway repository, patching out the explicit --install-scripts and using a local binary job script would be enough to indicate whether or not this would actually work.